packages/test/src/helpers-integration.ts

```diff
@@ -163,3 +180,98 @@ export function helpers(t: ExecutionContext<Context>): Helpers {
     },
   };
 }
+
+export interface HelpersTimeSkipping extends Helpers {
```

Comment:
I'd just add this to `Helpers` instead of having two different interfaces.
You want the entire suite to be able to run against either implementation (or an external server).

Reply:
Also, if you're extending the interface there's no need to redefine all of the methods.